### PR TITLE
User can select multiple drawing styles.

### PR DESCRIPTION
**Pull request by danielduffield**
Created a toolbar with an eraser tool, a size selector, and a set of colors to choose from, using new React class components PaintToolbar (which includes state for color and erasing) and SizeSelector.

### CODE REVIEW

**Comment by danielduffield** [comment]
con-canvas.herokuapp.com

**Comment by danielduffield** [comment]
concanvas-issue-7

File: client/utils/canvas.js [hunk 1]
```diff
@@ -61,11 +74,13 @@ export default class Canvas extends React.Component {
     const canvasData = await response.json()
     img.src = canvasData.saved
     this.lastSaved = canvasData.saved
-
-    this.unsavedData.forEach(mark => this.paintEvent(mark.x, mark.y, mark.prevX, mark.prevY))
+    console.log(this.unsavedData)
```
Comment (thebearingedge) [inline client/utils/canvas.js]: In general, never check in debugging logs like this.

File: client/utils/paint-toolbar.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          })}
+        </div>
+        <div id="eraser-label" className="toolbar-label">Eraser</div>
+        <div id="line-width-label" className="toolbar-label">Size</div>
+        <div id="current-color-label" className="toolbar-label">Current</div>
+      </div>
+    )
+  }
+}
+// <div id="line-width-tool" className="toolbar-module"></div>
```
Comment (thebearingedge) [inline client/utils/paint-toolbar.js]: Do you need this comment?

File: server/public/styles.css [hunk 3]
```diff
@@ -21,3 +21,100 @@
...
+
+.container {
+  text-align: center;
+}
+
+#my-canvas {
+  border: 2px solid black;
+}
+
+#paint-tools {
```
Comment (thebearingedge) [inline server/public/styles.css]: Is `styled-components` a good choice for your project? It's meant to replace this type of component-specific CSS.

**Review by thebearingedge — APPROVED** [review]
Great job, a few things to consider.

**Pull request merged**